### DISCUSSION TITLE
Wireshark dissector for TREL packets

### DISCUSSION THREAD

**Discussion by dounie** [openthread/openthread#8931] [Q&A] [answered]
I am currently trying to understand the interest of having a TREL link between two nodes instead of having two OTBR and using classical IPv6 routing mechanism. 

However, wireshark don't seem to parse TREL-UDP packets (while I can do it by hand for TREL header fields).
Is there an existing wireshark dissector to parse TREL packet, but also uncypher it ?
I would love it ! 
Thanks !

**Comment by abtink** [comment, marked as answer]
Thread spec chapter 14 (in particular section 14.1 introduction) provides some background and reason for TREL.

In short, it helps solve problems with Thread network partitions, in addition to allowing Thread network to benefits like higher throughput, channel capacity, coverage of other available IP-based technologies like WiFi on BRs.

About wireshark, I am not aware of any current dissector. It'd be cool if someone from community can contribute this.
